ocs_ci/ocs/cluster_load.py

```diff
@@ -82,7 +100,8 @@ def increase_load(self, rate=None, wait=True):
         new_args = [
             x for x in args if not x.startswith('--filesize=') and not x.startswith('--rate=')
         ]
-        new_args.append(f"--filesize={self.io_file_size}")
+        io_file_size = f"{self.pvc_size * 1000 - 200}M"
```

Comment:
I am out of the context here, can you just explain to me what this size is exactly above? Thanks

Reply:
This is the file size used for FIO. I set it to be according to the size of the PVC which is attached to the FIO pod, minus 200MB.